innersource-license.md

```diff
@@ -0,0 +1,85 @@
+## Title
+
+InnerSource License
+
+## Patlet
+
+Two companies that belong to the same group want to share software source code with each other but they are concerned about the implications in terms of legal liabilities or cross-company accounting.
+
+An **InnerSource License** provides a reusable legal framework for the inter-company agreement within the group, which opens up new collaboration options, and makes the rights and obligations of the involved companies explicit.
+
+## Problem
+
+When two or more companies within a group want to share code with each other, they need an agreement about the terms and often a legal contract. Creating such agreements on a per project basis takes effort and creates a barrier for sharing. i.e. a team/company might decide not to share their source code with another company in the group because it seems complicated.
+
+Barriers for sharing can lead to silos and duplication of effort in rebuilding similar solutions in multiple parts of the group.
+
+At the time of sharing the source code, it can not be reliably predicted what the value of sharing will be. If the activity of sharing requires significant effort (i.e. negotiating terms for the usage), the companies are less likely to do it, as they are concerned about the return on investment.
+
+## Context
+
+- A large group with many companies (subsidiaries) that want to share code. When the group gets larger, the value of this pattern increases.
+- These companies are legal entities with their own legal rights and obligations.
+- Multiple of these companies/subsidiaries are developing software, and are using services of the other companies. They have a motivation to contribute to each other’s source code.
+- A sufficient complexity of the organization and its organizational structure
+
+## Forces
+
+- **Level of effort** required to write formal agreements, especially if they need to take into account technical, legal, and business perspectives.
+- Enterprises have many **internal regulations,** any agreements have to comply with these regulations, e.g. security, privacy, procurement processes, etc. The volume of regulations can make it difficult to assess whether sharing software between two companies is compliant with these regulations, especially when there is no standard procedure.
```

Comment:
Here, the term 'Enterprise' just emerged from the scrubs. 
Could we align it with the other terms the patterns has or explain the difference in semantics to the others?

Reply:
Have replaced it with "Large groups of companies (enterprises)" for now.

It might be that we revisit these terms completely but that is discussed at length in other comments already. Hence closing this.